### PR TITLE
OoyalaApiAutomationHook

### PR DESCRIPTION
**Pull request by bindumadhav**

### CODE REVIEW

File: OoyalaAPISampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java [hunk 1]
```diff
@@ -100,6 +105,11 @@ public void update(Observable arg0, Object argN) {
       return;
     }
 
+    // Automation Hook: to write Notifications to a temporary file on the device/emulator
+    String text="Notification Received: " + arg1 + " - state: " + player.getState();
+    // Automation Hook: Write the event text along with event count to log file in sdcard if the log file exists
+    playbacklog.writeToSdcardLog(text);
+
     Log.d(TAG, "Notification Received: " + arg1 + " - state: " + player.getState());
```
Comment (brol1dev) [inline OoyalaAPISampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java]: I think the text variable can be reused for the Log here too. Please use it as `Log.d(TAG, text);`

**Review by brol1dev — APPROVED** [review]
Added one comment, although LGTM!

File: OoyalaAPISampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java (hunk 1 above)
Comment (bindumadhav) [reply OoyalaAPISampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java]: got it :) my bad . Will make changes in both PR's and raise a new request again

**Comment by bindumadhav** [comment]
Completed the changes based on CR comments

**Pull request merged**